pulp_rpm/app/advisory.py

```diff
@@ -136,10 +138,20 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
             to_remove.append(previous_advisory_pk)
             to_exclude.append(added_advisory_pk)
         else:
-            raise AdvisoryConflict(_('Incoming and existing advisories have the same id and '
-                                     'timestamp but different and intersecting package lists. '
-                                     'At least one of them is wrong. '
-                                     f'Advisory id: {previous_advisory.id}'))
+            if previous_updated_version != added_updated_version:
+                if previous_updated_version < added_updated_version:
```

Comment:
use is_previous_revision for comparison.
+1 to give is_previous_revision more generic name and update a docstring there